package-lock.json

```diff
@@ -4808,6 +4818,15 @@
                         "read-pkg": "^5.2.0",
                         "type-fest": "^0.8.1"
                     }
+                },
+                "semver": {
```

Comment:
😨

Reply:
Absolutely! I think that this problem is actually why semver was tripping in the first place--there was one version webpacked in, and another included as a dependency of the language client.

Reply:
Aha, that was _exactly_ it! And now fixed in #2626.